apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -39,6 +43,7 @@ export const BaseTicketForm: React.FC<ITicketFormProps> = (props) => {
     const UserInfoTitle = intl.formatMessage({ id: 'pages.condo.ticket.title.ClientInfo' })
     const TicketInfoTitle = intl.formatMessage({ id: 'pages.condo.ticket.title.TicketInfo' })
     const TicketPurposeTitle = intl.formatMessage({ id: 'TicketPurpose' })
+    const UploadedFilesLabel = intl.formatMessage({ id: 'component.uploadlist.AttachedFilesLabel' })
```

Comment:
Maybe it's better to keep variable names and keys conformed to each other, like `const AttachedFilesLabel` (to match key name)?

Reply:
Done